ncbi-amrfinderplus/3.10.36/Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM ubuntu:jammy as app
+
+ARG AMRFINDER_VER="3.10.36"
+
+LABEL base.image="ubuntu:jammy"
+LABEL dockerfile.version="1"
+LABEL software="NCBI AMRFinderPlus"
+LABEL software.version="3.10.5"
+LABEL description="NCBI resistance gene detection tool"
+LABEL website="https://github.com/ncbi/amr"
+LABEL maintainer1="Kelsey Florek"
+LABEL maintainer1.email="kelsey.florek@slh.wisc.edu"
+LABEL maintainer2="Curtis Kapsak"
+LABEL maintainer2.email="kapsakcj@gmail.com"
+LABEL maintainer3="Anders Goncalves da Silva"
+LABEL maintainer3.email="andersgs@gmail.com"
+LABEL maintainer4="Erin Young"
+LABEL maintainer4.meail="eriny@utah.gov"
+
+RUN apt-get update && apt-get install -y \
```

Comment:
`RUN apt-get update && apt-get install -y \` -> `RUN apt-get update && apt-get install -y --no-install-recommends \`

to save on a little bit of disk space. This was in the previous version's dockerfile

Reply:
I removed it because I was getting an error. There's probably a better way to get rid of that error, though.

Reply:
adding ca-certificates fixed it!